src/main/java/org/embulk/input/jira/JiraInputPlugin.java

```diff
@@ -172,7 +172,7 @@ public TaskReport run(final TaskSource taskSource,
     public ConfigDiff guess(final ConfigSource config)
     {
         // Reset columns in case already have or missing on configuration
-        config.set("columns", new ObjectMapper().createArrayNode());
```

Comment:
You may want to remove `import com.fasterxml.jackson.databind.ObjectMapper;` as well?